### PR TITLE
feat: Add Services page.

### PR DESCRIPTION
**Pull request by rut1303**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,12 +10,14 @@
     "@types/node": "^16.18.13",
     "@types/react": "^18.0.28",
     "@types/react-dom": "^18.0.11",
+    "@types/relay-runtime": "^14.1.8",
```
Comment (vaidehi27) [inline package.json:13]: What does this dependency do? why do we need it?

File: src/Pages/Services/constant.ts [hunk 2]
```diff
@@ -0,0 +1,138 @@
+export interface Service {
+    name: string;
+    description: string;
```
Comment (vaidehi27) [inline src/Pages/Services/constant.ts:3]:
```suggestion
    description1: string;
```

File: src/Pages/Services/index.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+setSelectedService(service);
+};
+
+return (
+<Container>
+<Header>Our Services</Header>
+<ServicesList>
+{services.map(service => (
+<ServiceItem key={service.name} onClick={() => handleServiceSelect(service)}>
+<ServiceName>{service.name}</ServiceName>
```
Comment (vaidehi27) [inline src/Pages/Services/index.tsx:28]: What is this? There is no formatting? I have already said that if the files are not formated, it should not allow to push PR. Who is working on that check?

File: src/Pages/Services/style.ts [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+`;
+
+export const ServiceItem = styled.li`
+  display: flex;
+  align-items: center;
+  padding: 1rem;
+  cursor: pointer;
+
+  &:hover {
+    background-color: #fc9e39;
```
Comment (vaidehi27) [inline src/Pages/Services/style.ts:29]: create a color constant instead here

File: package.json [hunk 5]
```diff
@@ -10,12 +10,14 @@
...
     "bootstrap": "^5.2.3",
     "react": "^18.2.0",
     "react-bootstrap": "^2.7.2",
     "react-dom": "^18.2.0",
     "react-scripts": "5.0.1",
-    "styled-components": "^5.3.6",
+    "react-typescript": "^1.0.0",
```
Comment (vaidehi27) [inline package.json:19]: why is this dependency needed?

**Review by vaidehi27 — COMMENTED** [review]
Why is your build failing?

File: src/Pages/Services/index.tsx (hunk 3 above)
Comment (rut1303) [reply src/Pages/Services/index.tsx:28]: I had some problem in installation of prettier ,I resolved it and update the files.

File: src/Pages/Services/style.ts (hunk 4 above)
Comment (rut1303) [reply src/Pages/Services/style.ts:29]: Ohk, I'll

Comment (rut1303) [reply src/Pages/Services/style.ts:29]: And update the file.

**Pull request closed without merging**